js/form-validation.js

```diff
@@ -0,0 +1,8 @@
+document.getElementById('contact').addEventListener('submit', (event) => {
+  const email = document.getElementById('email').value;
+  if (email.value !== email.toLowerCase()) {
+    event.preventDefault();
+    document.querySelector('.error').innerHTML = 'Your email address should be in lowercase';
+    event.target.reset();
+  }
+});
```

Comment:
@tresorsawasawa Your request is not clear enough can you please explain furthur?

Reply:
Hi @niicode 
I the email is in lowercase only the form will be sent normally but if it contains uppercase characters then it will show the error message. But in case the error is shown in both situations.